docs/scheduler/start_scheduler.rst

```diff
@@ -0,0 +1,39 @@
+Starting the Scheduler
+======================
+
+
+With docker
+-----------
+
+Installation process
+~~~~~~~~~~~~~~~~~~~~
+
+Docker will download worker image of syncmaster scheduler (same as backend image) & broker, and run them.
+Options can be set via ``.env`` file or ``environment`` section in ``docker-compose.yml``
+
+.. dropdown:: ``docker-compose.yml``
+
+    .. literalinclude:: ../../docker-compose.yml
```

Comment:
Could we highlight lines specific for a scheduler? Like this:
https://github.com/MobileTeleSystems/data-rentgen/blob/952dfe10261432985fa04d80e28e3fba25a51791/docs/reference/consumer/index.rst?plain=1#L30

Same for worker docs